performance_manager/lib/gtfs_static_table.py

```diff
@@ -0,0 +1,317 @@
+from typing import List, Optional
+
+import logging
+import pandas
+import numpy
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from .s3_utils import read_parquet, file_list_from_s3
+from .postgres_utils import MetadataLog, get_unprocessed_files
+from .postgres_schema import (
+    StaticFeedInfo,
+    StaticTrips,
+    StaticRoutes,
+    StaticStops,
+    StaticStopTimes,
+    StaticCalendar,
+)
+from .gtfs_utils import start_time_to_seconds
+
+
+def get_static_parquet_paths(table_type: str, feed_info_path: str) -> List[str]:
+    """
+    get static table parquet files from FEED_INFO path
+    """
+    springboard_bucket = "mbta-ctd-dataplatform-dev-springboard"
+    static_prefix = feed_info_path.replace("FEED_INFO", table_type)
+    static_prefix = static_prefix.replace(f"{springboard_bucket}/", "")
+    return list(
+        uri for uri in file_list_from_s3(springboard_bucket, static_prefix)
+    )
```

Comment:
The current structure of this is that everything gets loaded off `FEED_INFO` files. So for each specific table, the `FEED_INFO` prefix is mangled for each static table name. 

Not sure if this is something we want to go with, very open to discussing options.